Sources/Logging/Logging.swift

```diff
@@ -498,11 +498,34 @@ public struct MultiplexLogHandler: LogHandler {
     }
 }
 
+/// A wrapper to facilitate `print`-ing to stderr and stdio
+private struct FileOutputStream: TextOutputStream {
+    var file: UnsafeMutablePointer<FILE>
+
+    func write(_ string: String) {
+        string.withCString { ptr in
+            _ = fputs(ptr, file)
+        }
+    }
+}
+
 /// Ships with the logging module, really boring just prints something using the `print` function
-public struct StdoutLogHandler: LogHandler {
+public struct StdioLogHandler: LogHandler {
+
+    public enum Stream {
+        case stdout, stderr
+    }
+
     private let lock = Lock()
+    public let stream: Stream
+
+    public init(label: String) {
```

Comment:
this actor seems redundant given the other one has a default?

Reply:
To better express that and prevent the knee-jerk reaction you had, I think I'll remove the default from the other one, calling conventions will remain the same.